packages/browser/src/sdk.ts

```diff
@@ -1,8 +1,16 @@
 import { initAndBind } from '@sentry/core';
 import { getCurrentClient as shimGetCurrentClient } from '@sentry/shim';
+import { Integration } from '@sentry/types';
 import { BrowserOptions } from './backend';
 import { BrowserClient } from './client';
 
+import { OnError, OnUnhandledRejection } from './integrations';
+
+const DEFAULT_INTEGRATIONS: Integration[] = [
```

Comment:
Since we are not exporting this, I think we can move it directly into the call of `initAndBind`